src/components/UserDashboard/SavedQueries/SavedQueries.module.css

```diff
@@ -81,3 +81,14 @@
   font-size: 12px;
   color: rgb(52, 52, 52);
 }
+
+.spinner {
```

Comment:
same comment, why we need a spinner when the Card is supposed to handle that

Reply:
These cards are not antd cards